xrpl/models/transactions/credential_accept.py

```diff
@@ -0,0 +1,51 @@
+"""Model for CredentialAccept transaction type."""
+
+from dataclasses import dataclass, field
+from typing import Dict
+
+from typing_extensions import Self
+
+from xrpl.models.required import REQUIRED
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.models.transactions.types import TransactionType
+from xrpl.models.utils import (
+    KW_ONLY_DATACLASS,
+    _get_credential_type_error,
+    require_kwargs_on_init,
+)
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True, **KW_ONLY_DATACLASS)
+class CredentialAccept(Transaction):
+    """This transaction accepts a credential issued to the Account (i.e. the Account is
+    the Subject of the Credential object). The credential is not considered valid until
+    it has been transferred/accepted.
+    """
+
+    transaction_type: TransactionType = field(
+        default=TransactionType.CREDENTIAL_ACCEPT,
+        init=False,
+    )
+
+    account: str = REQUIRED  # type: ignore
+    """
+    The subject of the credential.
+    """
+    issuer: str = REQUIRED  # type: ignore
```

Comment:
```suggestion

    issuer: str = REQUIRED  # type: ignore
```

Reply:
I don't understand, what is the `diff` in your suggestion ?

Reply:
It's a newline for separation :)

Reply:
ah okay :)